library/alloc/src/vec.rs

```diff
@@ -1088,6 +1088,39 @@ impl<T> Vec<T> {
         }
     }
 
+    /// Tries to remove an element from the vector. If the element at `index` does not exist,
+    /// `None` is returned. Otherwise `Some(T)` is returned
+    /// 
+    /// # Examples
+    /// ```
+    /// let mut v = vec![1, 2, 3];
+    /// assert_eq!(v.remove(0), Some(1));
+    /// assert_eq!(v.remove(2), None);
+    /// ```
+    pub fn try_remove(&mut self, index: usize) -> Option<T> {
```

Comment:
Please open a tracking issue following the instructions in https://rustc-dev-guide.rust-lang.org/implementing_new_features.html?highlight=tracking,is#tracking-issues, then change `issue="none"` to the issue number.
```suggestion
    #![unstable(feature = "vec_try_remove", issue = "none")]
    pub fn try_remove(&mut self, index: usize) -> Option<T> {
```

Reply:
Would this: `#![unstable(feature = "vec_try_remove", issue = "77481")]` be the correct gate declaration?

Reply:
Yup, that would do it :)